src/_langs/en/updates/2015-07-22-access-ble-devices-with-web-bluetooth.markdown

```diff
@@ -0,0 +1,279 @@
+---
+
+layout: update
+published: true
+
+collection: updates
+category: chrome
+product: chrome
+type: news
+date: 2015-07-22
+
+title: "Access BLE devices with Web Bluetooth"
+description: "A Web API has been added to Chrome OS M45 that
+makes it possible for websites to discover and communicate with devices over
+the Bluetooth 4 wireless standard using GATT."
+article:
+  written_on: 2015-07-22
+  updated_on: 2015-07-22
+
+authors:
+  - beaufortfrancois
+tags:
+  - Bluetooth
+  - IoT
+  - WebBluetooth
+  - PhysicalWeb
+permalink: /updates/2015/07/access-ble-devices-with-web-bluetooth.html
+---
+
+How about I tell you websites could communicate with nearby Bluetooth devices
+in a secure and privacy-preserving way? This way, heart rate monitors, singing
+lightbulbs and [turtles](https://www.youtube.com/watch?v=1LV1Fk5ZXwA) could
+be managed directly from a website.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how works Bluetooth Low
+Energy (BLE) and Generic Attribute Profile (GATT).
+
+Even though the [Web Bluetooth API
+specification](https://webbluetoothcg.github.io/web-bluetooth/) is not
+finalized yet, the Chrome Team is actively looking for enthusiastic developers
+(I mean you) to try out this work-in-progress API and give
+[feedback](https://code.google.com/p/chromium/issues/entry?labels=Cr-Blink-Bluetooth).
```

Comment:
[feedback on the spec](https://github.com/WebBluetoothCG/web-bluetooth/issues) and [feedback on the implementation](https://code.google.com/p/chromium/issues/entry?labels=Cr-Blink-Bluetooth)